.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+Main.*
```

Comment:
let's better remove this
we shouldn't push class Main in this task, but we shouldn't change gitignore

Reply:
done